app/components/views/Info/index.tsx

```diff
@@ -146,6 +132,17 @@ export const Info: FC = () => {
                   ariaLabel={info.ariaLabel}
                   address={info.address}
                 />
+                {info.metamaskAriaLabel &&
+                  typeof props.provider?.provider?.request === "function" &&
+                  props.provider.provider.isMetaMask && (
```

Comment:
I realized some people have LP so we should rename the Pools to LP and add the metamask button to those as well.
Just need to add aria lables for those two
```suggestion
                {typeof props.provider?.provider?.request === "function" &&
                  props.provider.provider.isMetaMask && (
```